packages/lesswrong/server/resolvers/userResolvers.ts

```diff
@@ -273,7 +319,238 @@ addGraphQLResolvers({
       return updatedUser
     },
   },
+  // Statistics to include:
```

Comment:
@s-cheng This was a note to myself while I was working on it, but I thought it might be useful to leave it in while it's still in progress. I can remove it if you want